.github/workflows/create-dev-tag.yml

```diff
@@ -0,0 +1,69 @@
+name: Create Development Tag
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    - cron: '5 3 * * *'
+
+env:
+  GOWORK: off
+  GOPRIVATE: github.com/mesosphere
+  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  GH_ACCESS_TOKEN: ${{ secrets.MESOSPHERECI_USER_TOKEN }}
+
+jobs:
+  create-dev-tag:
+    runs-on:
+    - self-hosted
+    - small
+    strategy:
+      matrix:
+        branch:
```

Comment:
looking at this again, this will run on every branch when we cut release branches? I think we only want it to run on `main` if we are listing the release branches like this.

I think we can add an `if` statement to specify that this job should only run on main?